packages/core/src/diff/changes/type.ts

```diff
@@ -80,15 +120,15 @@ export function typeKindChangedFromMeta(args: TypeKindChangedChange) {
 }
 
 export function typeKindChanged(
-  oldType: GraphQLNamedType,
+  oldType: GraphQLNamedType | null,
```

Comment:
I think this event should also not be created when "oldType" is null... I will fix this.